File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/AfpKompensasjonstillegg.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.beregning.Ytelseskomponent;
import no.nav.domain.pensjon.regler.kode.YtelsekomponentTypeCti;

public class AfpKompensasjonstillegg extends Ytelseskomponent {

    private int referansebelop;
    private int reduksjonsbelop;
    private double forholdstallKompensasjonstillegg;


    public AfpKompensasjonstillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("AFP_KOMP_TILLEGG");
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/AfpKompensasjonstillegg.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent
import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

class AfpKompensasjonstillegg : Ytelseskomponent() {
    var referansebelop = 0
    var reduksjonsbelop = 0
    var forholdstallKompensasjonstillegg = 0.0

    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.AFP_KOMP_TILLEGG
}